### PR TITLE
Update de-DE.i18n.json

### PR DESCRIPTION
**Pull request by pixxelspotter**
Closes #ISSUE_NUMBER

### CODE REVIEW

File: src/i18n/de-DE.i18n.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "copy": "&Kopieren",
     "paste": "&Einfügen",
     "selectAll": "Alle auswählen",
-    "undo": "Wider&rufen",
+    "undo": "Wieder&rufen",
```
Comment (tassoevan) [inline src/i18n/de-DE.i18n.json:7]: I don't know German so much, but aren't you introducing a typo here?

Comment (pixxelspotter) [reply src/i18n/de-DE.i18n.json:7]: Sure, sorry my mistake, didn't know what io was thinking when i saw it. You can delete my Pull request

Comment (tassoevan) [reply src/i18n/de-DE.i18n.json:7]: No problem! 👍

**Comment by tassoevan** [comment]
Closing at request of @pixxelspotter

**Pull request closed without merging**